src/App.css

```diff
@@ -0,0 +1,257 @@
+@import url("http://fonts.googleapis.com/css?family=Lato:400,700");
+
+:root {
+  --pink: #f03f83;
+  --dark-pink: #ec4c8a;
+  --light-pink: #fc5193;
+  --T: #51fcda;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+  color: #fff;
+}
+
+body {
+  background-color: var(--pink);
+}
```

Comment:
Great design @TechnoAndy 🤝

- [ ] As specified in the project requirements, kindly follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including typography and layout.
- Your design
![your-design](https://user-images.githubusercontent.com/63917725/206808615-c8bbbc00-c699-4e7f-8432-b818a895b2a3.PNG)
- Given design
![given-design](https://user-images.githubusercontent.com/63917725/206808653-208c2f28-876a-4b34-8fe4-cdf831f4141e.PNG)